server/sonar-db-dao/src/main/java/org/sonar/db/qualitygate/QualityGateDao.java

```diff
@@ -55,6 +55,8 @@ public void update(QualityGateDto qGate, DbSession session) {
     mapper(session).update(qGate.setUpdatedAt(new Date()));
   }
 
+  public void ensureOneBuiltInQualityGate(DbSession dbSession, String builtInName) { mapper(dbSession).ensureOneBuiltInQualityGate(builtInName); }
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this line start at column 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)